Source/WebCore/css/html.css

```diff
@@ -226,8 +217,7 @@ h2 {
     font-size: 1.5em;
     margin-block-start: 0.83__qem;
     margin-block-end: 0.83em;
```

Comment:
```suggestion
    margin-block: 0.83__qem 0.83em;
```

Reply:
I didn't done because of `__qem` (quirky em) but happy to do.

Reply:
I would do it since it's more consistent that way

Reply:
Yep - just doing changes locally, will push it shortly. :-)

Reply:
@nt1m - done for all possible cases IMO. :-)